Makefile.am

```diff
@@ -29,6 +29,8 @@ noinst_DATA = dovecot-config
 
 nodist_pkginclude_HEADERS = config.h
 
+distuninstallcheck_listfiles = find . -type f \! -name '*.conf' -print
```

Comment:
We only have one dovecot.conf now. Seems excessive to use find?

Reply:
In theory, the variable lists everything else. I just copied the model from elsewhere.

Reply:
it's fine.